eye2bids/edf2bids.py

```diff
@@ -188,12 +196,22 @@ def _has_validation(df: pd.DataFrame) -> bool:
 def _extract_MaximalCalibrationError(df: pd.DataFrame) -> list[float]:
     if not _has_validation(df):
         return []
+    if _extract_RecordedEye(df) == "Both" and _extract_CalibrationCount(df) > 1:
+        return np.array_split(
+            (np.array(_validations(df)[[11]]).astype(float).tolist()),
+            _extract_CalibrationCount(df),
+        )
```

Comment:
double check but I think that we could do something like for this function and for `_extract_AverageCalibrationError`

```suggestion
        return np.array_split(
            np.array(_validations(df)[[11]]),
            _extract_CalibrationCount(df),
        ).astype(float).tolist()
```

also because it'd make sure that the returned type is always the same (this is the error you are getting for pre-commit in CI

Reply:
Yes, sounds good!

Reply:
I tested and modified the functions for average and max calibration error as suggested and pushed it to this pull request